Protocol.md

```diff
@@ -395,10 +396,10 @@ Frame Contents
                                   Data
 ```
 
-* __Frame Type__: (16) 0x03
-* __Flags__:
+* __Frame Type__: (6 bits = max value 63) 0x03
+* __Flags__: (10 bits)
      * (__R__)espond with KEEPALIVE or not
-* __Last Received Position__: (64) Resume Last Received Position (optional. Set to all 0s when not supported.)
+* __Last Received Position__: (64 bits = max value 2^63-1) Signed positive Long of Resume Last Received Position (optional. Set to all 0s when not supported.)
```

Comment:
Is there a value to specify the max value? It feels not very useful in this case. How about we omit the max value everywhere?